ide/forms.py

```diff
@@ -0,0 +1,25 @@
+from allauth.account.forms import SignupForm
```

Comment:
Please re-implement the signup form as react components and isn't there any better way to implement signup feature. why do we a forms.py. Can you do some more research and come up with some better approach. For reference you can see the implementation of EvalAI authentication.

Reply:
I see. Let me try reimplementing this in a better way.